protocols/direct-funding_test.go

```diff
@@ -1,14 +1,88 @@
 package protocols
 
 import (
+	"math/big"
 	"testing"
 
+	"github.com/ethereum/go-ethereum/common"
 	"github.com/statechannels/go-nitro/channel/state"
+	"github.com/statechannels/go-nitro/types"
 )
 
+// TestNew tests the constructor using a TestState fixture
 func TestNew(t *testing.T) {
 	_, err := NewDirectFundingObjectiveState(state.TestState, state.TestState.Participants[0])
 	if err != nil {
```

Comment:
if you find yourselves getting annoyed by go's built-in very minimal testing assertions, I recommend checking out https://github.com/stretchr/testify . I can source other recommendations from team if you need (I think I remember maybe something that is more lightweight?)

Reply:
If I read your code right,  `NewDirectFundingObjectiveState` errors if TestState.ChannelID() errors. I wonder if you want to make an InvalidTestState and test both conditions.